TWLight/templates/base.html

```diff
@@ -254,6 +254,12 @@
               {% trans "Feedback" %}
             </a>
           </li>
+          <li>
+            <a href="https://translatewiki.net/wiki/Translating:Wikipedia_Library_Card_Platform">
```

Comment:
I'd recommend changing this into the unlocalized project name, e.g. `translatewiki.net`
```suggestion
            <a href="https://translatewiki.net/wiki/Translating:Wikipedia_Library_Card_Platform">translatewiki.net</a>
```